packages/protocol/contracts/tokenvault/ERC1155Vault.sol

```diff
@@ -144,7 +144,9 @@ contract ERC1155Vault is BaseNFTVault, ERC1155ReceiverUpgradeable {
             }
         }
 
-        _to.sendEther(msg.value);
+        if (msg.value > 0) {
```

Comment:
I think we should still allow sending 0 ETH to message.owner, in case this addresss "receive" function is not empty.

Same for other changes.

Reply:
I guess it may be used as some kind of trigger that a bridge transaction was just done (receive function could figure that out by checking `msg.sender`, so maybe useful in some bridge specific contracts?) but otherwise there is no extra context what happened.

Either way fine for me, but if we do add the check I would do the `if (msg.value > 0) {` inside `sendEther`, it looks messy to see it everywhere.

Reply:
Sending a zero Ether to an address is a valid use case, I don't think our bridge should disable it. I think we should just make the invocation without checking if msg.value is zero. Calling a default receive function is very cheap with 0 ether.